src/routes.json

```diff
@@ -187,6 +187,12 @@
         "title": "Search"
     },
     {
+        "name": "search-redirect",
+        "pattern": "^/search/?$",
+        "routeAlias": "/search(?!/ajax)",
```

Comment:
we can leave the route alias as `/search` (like in `"search"`) above. The reason that explore is `"/explore(?!/ajax)"` has to do with some other sub-urls that still need to go to scratchr2